### PR TITLE
update view evaluation header dialogs

### PR DESCRIPTION
**Pull request by emmaacha**
Ticket ID: [View Eval Header - Use new dialog component #606](https://github.com/nerubia/kh360-react/issues/606)

### CODE REVIEW

File: src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx [hunk 1]
```diff
@@ -258,72 +258,58 @@ export const ViewEvaluationHeader = () => {
...
-        <Dialog.Actions>
-          <Button variant='primaryOutline' onClick={toggleReopenDialog}>
-            No
-          </Button>
-          <Button variant='primary' onClick={handleReopen} loading={loading === Loading.Pending}>
-            Yes
-          </Button>
-        </Dialog.Actions>
-      </Dialog>
+      <CustomDialog
```
Comment (gborgonia-ir) [inline src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx:261]: showSubmitButton and showCloseButton props not passed, same for other dialogs

Comment (emmaacha) [reply src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx:261]: I set default values for `showSubmitButton` and `showCloseButton` to true inside the CustomDialog component so the buttons would be visible by default. Is that okay?

Comment (gborgonia-ir) [reply src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx:261]: sorry didnt notice, thanks catt! Approved

Comment (emmaacha) [reply src/features/admin/evaluation-administrations/[id]/view-evaluation-header.tsx:261]: Alright, will re-request review ate thanks so much!

**Review by gborgonia-ir — APPROVED** [review]

**Review by chadiegilnerubia — APPROVED** [review]

**Pull request merged**